python/dgl/transform.py

```diff
@@ -291,7 +291,7 @@ def reverse(g, share_ndata=False, share_edata=False):
             [2.],
             [3.]])
     """
-    assert not isinstance(g, BatchedDGLGraph), \
+    assert g.batch_size == 1, \
```

Comment:
Could support instead? just set the same `batch_num_nodes` and `batch_num_edges`.

Reply:
DONE.